core/src/main/java/org/apache/accumulo/core/metadata/schema/MetadataScanner.java

```diff
@@ -62,10 +63,11 @@
     TableOptions from(Connector conn);
   }
 
-  public interface TableOptions extends RangeOptions {
-    /**
-     * Optionally set a table name, defaults to {@value MetadataTable#NAME}
-     */
+  public interface TableOptions {
```

Comment:
Does removing `extends RangeOptions` effectively make this a required parameter?

Reply:
yes